balancer/balancer.go

```diff
@@ -353,18 +353,20 @@ var ErrBadResolverState = errors.New("bad resolver state")
 //
 // It's not thread safe.
 type ConnectivityStateEvaluator struct {
-	numReady      uint64 // Number of addrConns in ready state.
-	numConnecting uint64 // Number of addrConns in connecting state.
+	numReady            uint64 // Number of addrConns in ready state.
```

Comment:
This reflects grpc's general state aggregation policy.  There are other ways to manage subconns, but this is the "standard" philosophy if SubConns are considered functionally equivalent.  I think it makes sense to be here, and moving it at this point would cause pain for our users, so I'd be opposed to that.